src/containers/calculation.js

```diff
@@ -256,7 +256,7 @@ function mapStateToProps(state, ownProps) {
   }
 
   const config = selectors.configuration.getConfiguration(state);
-  if (isNil(config.features)) {
+  if (!isNil(config.features) && !isNil(config.features)) {
```

Comment:
I don't understand, why is this necessary? The same statement is repeated twice and && does nothing as both times it should evaluate to the same thing shouldn't it?

Reply:
That is was it known as a typo

Reply:
Makes sense, otherwise I was at a total loss, the new patch makes perfect sense :+1: